Jenkinsfile

```diff
@@ -20,15 +32,24 @@ pipeline {
         echo 'Checking out project...'
         checkout scm
         echo 'Building...'
-        withMaven(jdk: 'OpenJDK11', maven: 'Maven3.6', mavenSettingsConfig: 'hcomdata-artifactory-maven-settings') {
-          sh 'mvn clean deploy jacoco:report checkstyle:checkstyle'
+        sh 'mvn clean deploy jacoco:report checkstyle:checkstyle -s ${MVN_OSS_SETTINGS}'
```

Comment:
We should add spotbugs in here (can be a separate PR)

Reply:
I did try this, but got a maven error saying it couldn't be found. Will make this a separate PR.

Reply:
Yes, you need to use the latest oss-parent (release this morning!) to get it.